build-script.py

```diff
@@ -187,7 +187,7 @@ def build_swiftsyntax(swift_build_exec, swiftc_exec, build_dir, build_test_util,
 ## Testing
 
 def run_tests(swift_test_exec, build_dir, release, swift_build_exec,
-              filecheck_exec, swiftc_exec, swift_syntax_test_exec, verbose):
+              filecheck_exec, swiftc_exec, swiftc_exec_unreal, swift_syntax_test_exec, verbose):
```

Comment:
What does "unreal" mean here?

Reply:
it's the opposite of 'realpath' in the script.